src/plugins.c

```diff
@@ -830,25 +1011,80 @@ static gboolean check_plugin_path(const gchar *fname)
 }
 
 
+/* Retuns NULL if this ain't a plugin,
+ * otherwise it returns the appropriate PluginProxy instance to load it */
+static PluginProxy* is_plugin(const gchar *file)
+{
+	GList *node;
+	const gchar *ext;
+
+	/* extract file extension to avoid g_str_has_suffix() in the loop */
+	ext = (const gchar *)strrchr(file, '.');
+	if (ext == NULL)
+		return FALSE;
+	/* ensure the dot is really part of the filename */
+	else if (strchr(ext, G_DIR_SEPARATOR) != NULL)
+		return FALSE;
```

Comment:
Doesn't seem to be true if I can read it (quickly) correctly: `load_active_plugins()` read `active_plugins_pref` directly, which was loaded from the config file.  There the path is obviously absolute (and it's the case in my local Geany's config), as it's checked for existence.
So AFAICT I could at least force a `\` in the path by manually editing the config file.